### PR TITLE
Add nozzle sizes for Agile printer

### PR DESCRIPTION
**Pull request by robintiger**

### CODE REVIEW

**Comment by ChrisTerBeke** [comment]
Could you describe in the PR title what this change actually does?

**Comment by robintiger** [comment]
> different type of nozzle for Agile 3D printer recently add "agile.def.json"

https://github.com/Ultimaker/Cura/pull/4803

**Comment by nallath** [comment]
For our reference; CURA-5968

File: resources/variants/agile_0.2.inst.cfg [hunk 1]
```diff
@@ -0,0 +1,14 @@
+[general]
+name = 0.2 mm
+version = 4
+definition = agile
```
Comment (LipuFei) [inline resources/variants/agile_0.2.inst.cfg]: The definition profile now is named as `zyyx_agile.def.json`. This needs to be changed to `zyyx_agile`.

File: resources/variants/agile_0.4.inst.cfg [hunk 2]
```diff
@@ -0,0 +1,14 @@
+[general]
+name = 0.4 mm
+version = 4
+definition = agile
```
Comment (LipuFei) [inline resources/variants/agile_0.4.inst.cfg]: Same as above, this needs to be changed to `zyyx_agile`.

File: resources/variants/agile_0.6.inst.cfg [hunk 3]
```diff
@@ -0,0 +1,14 @@
+[general]
+name = 0.6 mm
+version = 4
+definition = agile
```
Comment (LipuFei) [inline resources/variants/agile_0.6.inst.cfg]: Same as above, this needs to be changed to `zyyx_agile`.

File: resources/variants/agile_0.8.inst.cfg [hunk 4]
```diff
@@ -0,0 +1,14 @@
+[general]
+name = 0.8 mm
+version = 4
+definition = agile
```
Comment (LipuFei) [inline resources/variants/agile_0.8.inst.cfg]: Same as above, this needs to be changed to `zyyx_agile`.

File: resources/variants/agile_0.2.inst.cfg [hunk 5]
```diff
@@ -0,0 +1,14 @@
...
+
+[metadata]
+author = zekes
+setting_version = 4
```
Comment (LipuFei) [inline resources/variants/agile_0.2.inst.cfg]: `setting_version` not is `7`

File: resources/variants/agile_0.4.inst.cfg [hunk 6]
```diff
@@ -0,0 +1,14 @@
...
+
+[metadata]
+author = zekes
+setting_version = 4
```
Comment (LipuFei) [inline resources/variants/agile_0.4.inst.cfg]: `setting_version` not is `7`

File: resources/variants/agile_0.6.inst.cfg [hunk 7]
```diff
@@ -0,0 +1,14 @@
...
+
+[metadata]
+author = zekes
+setting_version = 4
```
Comment (LipuFei) [inline resources/variants/agile_0.6.inst.cfg]: `setting_version` not is `7`

File: resources/variants/agile_0.8.inst.cfg [hunk 8]
```diff
@@ -0,0 +1,14 @@
...
+
+[metadata]
+author = zekes
+setting_version = 4
```
Comment (LipuFei) [inline resources/variants/agile_0.8.inst.cfg]: `setting_version` not is `7`

File: resources/variants/agile_0.6.inst.cfg [hunk 9]
```diff
@@ -0,0 +1,14 @@
...
+type = variant
+hardware_type = nozzle
+
+[values]
+machine_nozzle_size = 0.4
```
Comment (LipuFei) [inline resources/variants/agile_0.6.inst.cfg]: I think this is supposed to be `0.6`.

**Comment by fieldOfView** [comment]
NB: this is going to lead to people complaining they can no longer set *custom* nozzle sizes.

**Comment by nallath** [comment]
Since no action was taken on this by the original author, I'm closing this.

**Pull request closed without merging**